dolphin/api/v1/storages.py

```diff
@@ -123,8 +132,31 @@ def create(self, req, body):
 
         return storage_view.build_storage(storage)
 
+    @validation.schema(schema_storages.update)
     def update(self, req, id, body):
-        return dict(name="Storage 4")
+        """Update storage access information."""
+        ctx = req.environ.get('dolphin.context')
+        try:
+            # Get existing access_info from DB
+            access_info_dict = db.access_info_get(ctx, id)
+            # Merge fields from request body to access_info
+            update_access_info(access_info_dict, body)
```

Comment:
the object return from db is not a dict. but you can update like this:
```
 access_info = db.access_info_get(ctx, id)
access_info.update(body)
```

and too many code comments, some of them are not required.

Reply:
changed to dict in update_access_info() function, also combined some comments